src/signals/incident-management/containers/Dashboard/components/BarChart/BarChart.tsx

```diff
@@ -13,7 +13,7 @@ export const BarChart = () => {
   return (
     <Wrapper>
       <ModuleTitle
-        title="Openstaande meldingen tot en met vandaag"
+        title="Openstaande meldingen tot en met vandaag "
         amount={nrOfIncidents}
```

Comment:
This does not work as expected. On a closer look I see that the dummy data in the ModuleTitle component is still there. You need to remove the dummy data and change it to the amount prop. Then you also should need to add a space in the title here. That should be covered in the title component.

Reply:
Changed it.